src/Widgets/TaskRow.vala

```diff
@@ -91,23 +88,53 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
         summary_entry_context.add_class (Gtk.STYLE_CLASS_FLAT);
         summary_entry_context.add_provider (taskrow_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
-        var due_image = new Gtk.Image.from_icon_name ("office-calendar-symbolic", Gtk.IconSize.BUTTON);
+        due_datetime_popover = new Tasks.DateTimePopover () {
+            icon = "office-calendar-symbolic",
+            placeholder = _("Set Due")
```

Comment:
This should probably be internal to DateTimePopover right? Like there's not a example where some instance of DateTimePopover will set something different